audits/README.md

```diff
@@ -0,0 +1,5 @@
+# Audits
+
+| Date          | Commit                                                                                     | Auditor      | Scope                | Links                                                       |
+| ------------- | ------------------------------------------------------------------------------------------ | ------------ | -------------------- | ----------------------------------------------------------- |
+| April 2025    | [`18db32c`](https://github.com/openzeppelin/openzeppelin-community-contracts/tree/18db32c) | OpenZeppelin | (see report))        | [🔗](./2015-04-18db32c.pdf)                                 |
```

Comment:
```suggestion
| April 2025    | [`18db32c`](https://github.com/openzeppelin/openzeppelin-community-contracts/tree/18db32c) | OpenZeppelin | (see report)        | [🔗](./2015-04-18db32c.pdf)                                 |
```

Do we need this column at all? Perhaps I'd do:

```suggestion
| April 2025    | [`18db32c`](https://github.com/openzeppelin/openzeppelin-community-contracts/tree/18db32c) | OpenZeppelin | vanilla-v5.3 changes        | [🔗](./2015-04-18db32c.pdf)                                 |
```

Or similar. The idea is to tie the audit to the 5.3 release of the vanilla contracts. Alternatively I'd remove the column altogether

Reply:
"vanilla-v5.3 changes" doesn't really make sens to me. I'd remove that column if you think its better.

Here the "real" answer is:
![Capture d’écran du 2025-04-07 15-36-43](https://github.com/user-attachments/assets/d7bde1b0-aae2-4d33-b9d9-099042bb1b01)

Reply:
I'd like having this column, with the "(see report)" mention, just so that its instantly clear that not everything is covered.

Reply:
Yes. We can remove the collumn and add a disclaimer above the table that says that all audits in this repo are partial audits, and that the scope of each one can be looked up in the reports